server/models/Contest.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const contestSchema = new Schema({
```

Comment:
Would like to have a list of urls as another attribute so that they keep track of whatever images the contest maker liked